modules/parser/src/parsers/parse-timeslice-data-v2.js

```diff
@@ -110,6 +111,10 @@ function parseStreamSets(streamSets, timestamp, convertPrimitive) {
         timeSeries.push(...streamSet.time_series);
       }
     }
+
+    if (streamSet.no_data_streams && noDataStreams) {
```

Comment:
Looks like `noDataStreams` is always defined?

Reply:
ah correct, left over code from earlier attempt